### DISCUSSION TITLE
Adding more text than title in portfolio widget

### DISCUSSION THREAD

**Discussion by 4bluedog** [elementor/elementor#18797] [Developers Community]
Hi,
I would like to add more text to the portfolio widget item overlay (custom field or at least description). Now there is only the title avalible. 
I've done some experiments with a custom Ele skin and custom fields, but can't make it look like a portfolio widget. The text should only be visible when you hover the cursor over the image, and the images should have square proportions. I would be grateful for your suggestions. In the link there is a comparison.

[portfolio - custom Ele skin comparsion](https://www.grebski.com.pl/projekty/)

**Comment by Gt-tuo** [comment]
I’m looking forward to an answer to this question, too. I also need to add the contents of a custom field to the overlay as well as icons to the category filter buttons. I managed to do both in a custom plugin with a copy of the portfolio-widget, though the result is missing the needed JS event handlers for the portfolio grid and the filters to work, although I use the same code as the portfolio-widget, and use the same css class names (in regard to JQuery selectors).
